QuizApp/QuizApp/Coordinator.swift

```diff
@@ -39,10 +31,7 @@ class Coordinator: CoordinatorProtocol {
     }
 
     private func makeQuizViewController() -> UINavigationController {
-        let quizViewController = QuizViewController(
-            viewModel: QuizViewModel(
-                coordinator: self,
-                quizUseCase: appDependencies.quizUseCase))
+        let quizViewController = Container.quizViewController()
         let quizNavigationController = UINavigationController(rootViewController: quizViewController)
```

Comment:
Please "unwrap" this VC from NavCon. In other words, you do not have to create additional NavCon.